msbuild.ps1

```diff
@@ -0,0 +1,12 @@
+echo test
+$PSScriptRoot = Split-Path -Parent -Path $MyInvocation.MyCommand.Definition
+$SDK_DIR = "$PSScriptRoot/.dotnet/sdk/1.0.1/"
```

Comment:
I would expect this to be broken. We install the 1.0.4 version of SDK.

Reply:
On second thought (after looking at the tasks.json), we *really* don't need this script.